lib/create-version-from-ingestion/get-version-languages.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const path = require('path');
+
+/**
+ * Get languages for the version, falling back through different manifests
+ *
+ * @param {Object} origamiManifest
+ * @param {Object} bowerManifest []
+ * @param {Object} packageManifest []
+ * @param {String} tag - The Github tag for this version
+ * @param {String} owner - The Github organisation for this version
+ * @param {String} repo - The Github repository name for this version
+ * @param {GitHubClient} githubClient - The app's Github client wrapper
+ * @returns
+ */
+module.exports = async function getVersionLanguages(
+    origamiManifest,
+    bowerManifest,
+    packageManifest,
+    tag,
+    owner,
+    repo,
+    githubClient
+) {
+    origamiManifest = origamiManifest || {};
+    bowerManifest = bowerManifest || {};
+    packageManifest = packageManifest || {};
+
+    const mainPaths = [];
+
+    const component = ['module', 'component'].includes(
+        origamiManifest.origamiType
+    );
+    const specV1 = origamiManifest.origamiVersion === 1;
+    const specV1Component = specV1 && component;
+
+    // Bower: old spec v1 projects are the most likely to be misconfigured,
+    // if there is a bower.json check it.
+    if (bowerManifest.main) {
+        mainPaths.push(...bowerManifest.main.flatMap(a => a));
```

Comment:
Suggestion:
This can be rewritten using `flat()` instead of `flatMap(a => a)`

```suggestion
        mainPaths.push(...bowerManifest.main.flat());
```

Reply:
I didn't know that method exists, thanks!